### ISSUE TITLE
Bug with the Tome of Perita

### ISSUE THREAD

**Issue by ThatGrlSteph** [WayofTime/BloodMagic#2060]
#### Issue Description:
I dont know how to get in touch with the author, but it is an exploit technically.

#### What happens:
This book is used to store xp, the skill tree in game on MMC4 V27 Forge, is based on combat xp only.
so normal xp will not let you level up the skill tree. But if you use this book to store your normal xp and then take it back out, it turns it into combat xp that you can use towards the skill tree. Big way to cheat the skill tree and level up fast with no work.

#### What you expected to happen:
normal xp to stay normal xp.

#### Steps to reproduce:

1. well i banned the item from my server but id hope it can be fixed.
2. 
3. 
...

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: 1.20.1-3.3.3-45
- Minecraft: Medieval MC 4 1.20.1
- Forge: Forge

**Comment by wrincewind** [comment]
I'd suggest raising a bug with Medieval MC - https://github.com/LunaPixelStudios/Medieval-MC/issues 

I'll be honest, i'm just here to write manuals, but as far as i was aware, an XP orb is just an XP orb, it doesn't track where it came from. Presumably Medieval MC is tracking these orbs somehow and it's getting assigned.... maybe to the player? 

It might be worth checking other things that store XP, like the memory stick from PneumaticCraft or the memory crystal from Thermal Foundation.

**Comment by ThatGrlSteph** [comment]
yeah in MMC4 theres 2 types of xp, normal xp that get from quest/furnaces/trading with villagers, and then xp you get from killing mobs which is combat xp. Your not supposed to use normal xp on the skill tree or that would make you complete it wayyyy to fast. 

I was on the blood magic mob discord and they told me to come tell you XD lol
